### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Timost**
Hi !
First of all thank you for working on this much needed azure storage layer.

I think the readme is not correct for the Gen 2 details. I think it should pass the `AzureBlobFileSystem` class instead of the `AzureDatalakeFileSystem`.

If that is correct. Here is the PR fixing the readme.

### CODE REVIEW

**Comment by martindurant** [comment]
Thank you for picking up on this.
After the coming change in fsspec, this step should not be necessary at all, so I expect this PR to be superseded ( cc @TomAugspurger )

(note that the failure is due to the implementation of linting)

**Comment by Timost** [comment]
Thanks for the quick reply ! Regarding the upcoming change in fsspec. Do you mean this PR: intake/filesystem_spec#206 ?

**Comment by martindurant** [comment]
That is an issue yet to be implemented, and that would work too, but I actually mean directly adding the implementation classes here to fsspec.registry.known_implementations

**Comment by Timost** [comment]
Oh ok, I couldn't find the related PR.! Are you interested in this PR then ? If yes, do you want me to edit the setup.py file for the linting to pass ?

**Comment by martindurant** [comment]
Yes, that PR to fsspec should be made.
Following that, the lines here concerning the fsspec registry should simply be removed, they will not be necessary. Regarding linting, yes, we should just accept what the linter says is the right format (I don't know why it changed).

**Comment by hayesgb** [comment]
Cross-referencing with this [PR.](https://github.com/intake/filesystem_spec/pull/209)

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ To use the Gen2 filesystem:
 ```
 import dask.dataframe as dd
 from fsspec.registry import known_implementations
-known_implementations['abfs'] = {'class': 'adlfs.AzureDatalakeFileSystem'}
+known_implementations['abfs'] = {'class': 'adlfs.AzureBlobFileSystem'}
```
Comment (martindurant) [inline README.md]: This shouldn't be needed anymore, right?

**Comment by hayesgb** [comment]
Correct — once the protocols are in the fsspec known_protocols registry, we can remove this from the adlfs Readme.

**Comment by Timost** [comment]
Ok I guess when intake/filesystem_spec#209 is released, this PR can be closed ? Thank you for adding this support !

**Comment by Timost** [comment]
I'm closing this PR. The ffspec `known_protocols` registry no longer needs patching as `adl` and `abfs` are now there in fsspec.

**Pull request closed without merging**